app/views/admin/delete_collection/form.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= turbo_frame_tag 'admin-card' do %>
+  <% if @collection.works.empty? %>
+    <%= render Elements::CardComponent.new(classes: 'admin-card') do |component| %>
+      <% component.with_body do %>
+        <%= form_with url: collection_admin_delete_path(druid: params[:collection_druid]), method: :delete do |form| %>
+          <%= render Elements::Forms::CheckboxComponent.new(form:, field_name: :confirm_decommissioned, label: 'I confirm this collection has been decommissioned in Argo and understand that this action cannot be undone.', required: true) %>
+          <%= render Elements::Forms::SubmitComponent.new(label: 'Delete', classes: 'mt-3') %>
+        <% end %>
+      <% end %>
+    <% end %>
+  <% else %>
+    <%= render Elements::AlertComponent.new(variant: :danger, value: t('messages.collection_not_empty')) %>
```

Comment:
Doesn't feel like this should be a danger.

Reply:
Cool, changed to :warning